static-metric/examples/metric_enum.rs

```diff
@@ -62,6 +62,10 @@ fn main() {
     assert_eq!(vec.with_label_values(&["post", "foo"]).get(), 1.0);
     assert_eq!(vec.with_label_values(&["delete", "bar"]).get(), 4.0);
 
+    // metric enums will expose an enum for type-safe `get()`.
+    static_counter_vec.get(Methods::post).foo.inc();
+    assert_eq!(static_counter_vec.post.foo.get(), 2.0);
```

Comment:
Can `static_counter.post.foo.inc()` also work?

Reply:
yes! This still work, see line 67. This PR enhances type safety for compile-time-unknown labels.

Reply:
:heart_eyes: Awesome.